inputfiles/idl/Fullscreen.widl

```diff
@@ -15,6 +15,6 @@ partial interface Document {
   attribute EventHandler onfullscreenerror;
 };
 
-partial interface DocumentOrShadowRoot {
+partial interface mixin DocumentOrShadowRoot {
```

Comment:
IDL files must not be hand-written. Currently the code takes care of partial interfaces incorrectly extends interface mixins, so IMO no need to change this manually.